scripts/release.ts

```diff
@@ -66,6 +67,64 @@ async function undoLastCommit() {
   console.log(c.green('Publish commit deleted.'));
 }
 
+const newDesignSystemVersion = (): string => {
```

Comment:
Because we're invoking this function for each child package, we're calling `npm version -w @cmsgov/design-system --json` more times than we probably need. What do you think about checking for whether a variable called `newDesignSystemVersion` is stored in memory and if not then firing off the npm version command?

Reply:
Good catch. Yeah I'll put into a var and then pass in as an argument.